packages/app/cypress/e2e/studio/helper.ts

```diff
@@ -1,3 +1,17 @@
+export function loadExperimentalStudio ({ specName = 'spec.cy.js', cliArgs = [''], enableCloudStudio = false, specSelector = 'data-cy-row' } = {}) {
```

Comment:
Initially, I thought this was supposed to load studio into studio mode, but I think it's name is meant to convey it's loading the `experimental-studio' project.

Suggest renaming to something like:
```suggestion
export function loadExperimentalStudioProject ({ specName = 'spec.cy.js', cliArgs = [''], enableCloudStudio = false, specSelector = 'data-cy-row' } = {}) {
```